props/saber_sa22c_buttons.h

```diff
@@ -0,0 +1,357 @@
+            current_volume_ += VOLUME * 0.10;
+            dynamic_mixer.set_volume(current_volume_);
+            beeper.Beep(0.5, 2000);
+            STDOUT.print("Current Volume: ");
+            STDOUT.println(dynamic_mixer.get_volume());
+          }
+          else{
+           beeper.Beep(0.5, 3000);
+          }
+        }
+#endif
+        if (!MODE_VOLUME){
+          aux_on_ = false;
+          On();
+        }
+	return true;
+
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_SHORT, MODE_OFF):
+#if NUM_BUTTONS == 2
+        if (MODE_VOLUME){
+          STDOUT.println("Volume Down");
+          if (dynamic_mixer.get_volume() > (0.10 * VOLUME)){
+            current_volume_ = dynamic_mixer.get_volume();
+            current_volume_ -= (VOLUME * 0.10) ;
+            dynamic_mixer.set_volume(current_volume_);
+            beeper.Beep(0.5, 2000);
+            STDOUT.print("Current Volume: ");
+            STDOUT.println(dynamic_mixer.get_volume());
+          }
+          else{
+            beeper.Beep(0.5, 1000);
+          }
+        }
+#endif
+#if NUM_BUTTONS == 3
+        if (MODE_VOLUME){
+          STDOUT.println("Volume up");
+          if (dynamic_mixer.get_volume() < VOLUME){
+            current_volume_ += VOLUME * 0.10;
+            dynamic_mixer.set_volume(current_volume_);
+            beeper.Beep(0.5, 2000);
+            STDOUT.print("Current Volume: ");
+            STDOUT.println(dynamic_mixer.get_volume());
+          }
+          else{
+            beeper.Beep(0.5, 3000);
+          }
+        }
+#endif
+        if (!MODE_VOLUME) {
+#ifdef DUAL_POWER_BUTTONS
+          aux_on_ = true;
+          On();
+#else
+      if (!MODE_VOLUME) {
+        next_preset();
+	  }
+
+#endif
+        }
+	return true;
+
+#if NUM_BUTTONS > 1
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_SHORT, MODE_ON):
+        SaberBase::DoBlast();
+        return true;
+#endif
+#if NUM_BUTTONS == 1
+      case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON | BUTTON_POWER):
+#ifndef DISABLE_COLOR_CHANGE
+        if (accel_.x < -0.15) {
+				  ToggleColorChangeMode();
+		} else {
+			  SaberBase::DoForce();
+		}
+#else
+        SaberBase::DoForce();
+#endif
+        return true;
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_LONG, MODE_ON):
+        StartOrStopTrack();
+        return true;
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_ON):
+        SaberBase::DoBlast();
+        return true;
+#endif
+      case EVENTID(BUTTON_POWER, EVENT_DOUBLE_CLICK, MODE_ON):
+        if (millis() - activated_ < 500) {
+          if (SetMute(true)) {
+            unmute_on_deactivation_ = true;
+          }
+        }
+#if NUM_BUTTONS > 1
+        else{
+          SaberBase::DoForce();
+        }
+#endif
+	return true;
+
+#if NUM_BUTTONS == 1
+        case EVENTID(BUTTON_POWER, EVENT_HELD_LONG, MODE_ON):
+        if (!SaberBase::Lockup()) {
+#ifndef DISABLE_COLOR_CHANGE
+	        if (SaberBase::GetColorChangeMode() != SaberBase::COLOR_CHANGE_MODE_NONE) {
```

Comment:
Indented too much.
Maybe try finding a text editor that understands C++ and have it do the indentation for you?